src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadoc1.java

```diff
@@ -0,0 +1,20 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = true
+forbiddenSummaryFragments = ^Returns the customer ID. This method returns.$
+period = .
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+public class InputSummaryJavadoc1 {
+
+    // violation 2 lines below 'd'
```

Comment:
>`d`

???

Reply:
And please create new issue on this Check to not do trim. User owns all symbols.
We do not make any opinion why trailing spaces are required. User should match on whole content.

Let's presume that user wants to violate trailing spaces in inline tags, he should be able to do with simple regexp